app/views/editions/comments.html.erb

```diff
@@ -14,7 +14,7 @@
       </a>
       <%= link_to(
             "View edition",
```

Comment:
Perhaps we need to change the link label to say "View changes"/ "View differences"?